README.md

```diff
@@ -7,13 +7,12 @@ This is the source for the [babeljs.io](https://babeljs.io) website; feel free t
 
 ### Setup
 
-Node: Check that Node is installed with version 10.19.0 and up. You can check this with node -v.
+Node: Check that Node is installed with version 20.5.0 and up. You can check this with `node -v`.
 
-Yarn: Make sure that Yarn 1 is installed with version >= 1.19.0.
+Yarn: Make sure that Yarn 4 is installed with version >= 4.0.1.
```

Comment:
Yarn 1.19 should also be fine, since it will automatically use the Yarn version installed in this repository. Anything higher than 1.19 is ok.